docs/uv/manager.rst

```diff
@@ -0,0 +1,59 @@
+Measurement Manager
+==========
+
+While reporters in `uv-metrics` handle values that have already been measured,
+the `MeasurementManager` feature allows you to schedule certain measurements to
+run at specified intervals (which may be different for each measurement), and
+automatically report the results.
+
+A `MeasurementManager` object holds a `reporter`, which it will use to report
+the measurements it makes.  To initialize a `MeasurementManager`, one passes
+a reporter and a dictionary containing items which are necessary to make the
+measurements (e.g., the test dataset), e.g.:
+
+.. code-block:: python
+
```

Comment:
show the import too, please!

Reply:
Thanks for the catch, this made me realize I wanted to add to the `__init__.py` file to make importing easier.  Now we can import (in a similar way to what you've done with reporter) by doing `import uv.manager as m` and calling `m.MeasurementManager()`

Reply:
I am not QUITE sure that this is the right package... `manager` is vague, while MeasurementManager` is clearer. But let's go with it and see how we feel down the road!